build.gradle

```diff
@@ -193,36 +176,25 @@ dependencies {
     implementation 'com.google.cloud:google-cloud-pubsub'
     implementation 'com.google.cloud:spring-cloud-gcp-starter-logging:4.9.0'
     implementation 'com.google.http-client:google-http-client'
-    implementation 'org.apache.commons:commons-dbcp2:2.7.0'     // For database connection support
-    implementation 'org.apache.commons:commons-lang3:3.11'
-    implementation 'org.apache.commons:commons-collections4:4.4'
+
+    implementation 'org.apache.commons:commons-dbcp2' // For database connection support
+    implementation 'org.apache.commons:commons-lang3'
+    implementation 'org.apache.commons:commons-collections4'
     implementation 'org.apache.directory.studio:org.apache.commons.io:2.4'
+    implementation 'org.apache.httpcomponents.client5:httpclient5'
+
     implementation 'org.liquibase:liquibase-core'
-    implementation 'org.postgresql:postgresql'          // Postgres jdbc driver
-    implementation 'ch.qos.logback:logback-core'
-    implementation 'ch.qos.logback:logback-classic'  // logback-classic natively implements slf4j api
 
     implementation 'org.codehaus.janino:janino'           // Provides if/else xml parsing for logback config
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'jakarta.validation:jakarta.validation-api'
     implementation 'jakarta.annotation:jakarta.annotation-api'
 
-    implementation 'org.apache.httpcomponents.client5:httpclient5'
-
     implementation 'io.swagger.core.v3:swagger-annotations'
     swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli'
 
     implementation 'org.springframework:spring-jdbc'
     implementation 'org.antlr:ST4:4.3'                          // String templating
-    implementation 'io.kubernetes:client-java:18.0.0'
```

Comment:
We did not use this direct dependency.  Maybe it was added previously as a means of mitigating a security vulnerability.  By pulling in latest TCL which pulls in (and uses) the latest k8s client, we can safely remove this.